tags/piracy.tag

```diff
@@ -5,6 +5,6 @@ type: text
 Cracked servers are not supported here and by setting your server as so, you will not receive support here any further.
 "Cracked" or "offline mode" servers basically disable any sort of Mojang authentication and allow people with non-genuine versions of the game to join. **This is software piracy and is illegal!**. By setting your server as so, you are condoning the use of piracy. 
```

Comment:
Offline mode itself isn't illegal, just pirating the game, maybe clarify that a bit

Reply:
Not sure how to clarify this, @DoctorMacc please advise.

Reply:
At the moment it implies that using offline mode is illegal

Reply:
How about, `Unless the server is behind an online mode BungeeCord instance, **This is software piracy and is illegal!**`

Yes, Circuit, I know it doesn't account for "testing".

Reply:
It just seems ride, even if people are pirated the game I don't see what that had to do with Geyser support